### ISSUE TITLE
Fix installation 200/201 handling; remove "invalid CERT" toast for GitHub sync

### ISSUE THREAD

**Issue by taylordowns2000** [OpenFn/Lightning#1042]
### Todo

- [x] Fix the installation path (the 200/201 funny business you detected just now)
- [x] Modify this **_instance level_** error to say to the end user "Sorry, it seems that the GitHub cert has not been properly configured for this instance of Lighting. Please contact the instance administrator."
- [ ] Manually send the exception to Sentry so that if this ever happens we see it immediately.

### More context:

Only the person who configures this instance—setting up the single PEM for the whole instance—needs to know about this error, and it should show up with the failing stack-trace on Sentry like other crashes. That devops person who installed/deployed lightning might not even have an in-app username. (They might... normally they'd be the single "SUPERUSER" for the instance.)

### Maybe even more:

Both of these don't seem like things that an end-user (no matter if they are viewer, editor, or admin on a project) could ever do anything about.

```ex
  defp installation_id_error do
    {:error,
     %{
       message:
         "Invalid installation ID, ensure to use the ID provided by Github"
     }}
  end

  defp invalid_pem_error do
    {:error,
     %{
       message:
         "Invalid Github PEM KEY, ensure to use the KEY provided by Github"
     }}
  end
```

In other words, they feel like they're bugs or mistakes during installation... but if that's the case we need them on Sentry, rather than handled through the app UI.

Example send to Sentry if you want to _not_ crash:
```ex
    if timeout? do
      Sentry.capture_message("Processor Timeout",
        level: "warning",
        message: error,
        extra: context,
        tags: %{type: "timeout"}
      )
    else
      Sentry.capture_exception(error,
        stacktrace: meta.stacktrace,
        message: error,
        error: error,
        extra: context,
        tags: %{type: "oban"}
      )
    end
```

**Comment by taylordowns2000** [comment]
back to backlog to make sure we get the whole error sent to sentry, not just the message: `Sentry.capture_exception`

**Comment by zacck** [comment]
@taylordowns2000  we don't have an exception at that point it's a failing pattern match https://github.com/OpenFn/Lightning/blob/1ec77331312536a2ab77baaaa29de693fa5d5bac/lib/lightning/version_control/github_client.ex#L128 
If you so wish I can however construct an exception and send that
